orttraining/orttraining/python/training/ortmodule/_custom_autograd_function.py

```diff
@@ -87,6 +87,4 @@ def enable_custom_autograd_support(to_enable=True):
 from onnxruntime.training import ortmodule  # noqa: E402
 
 # Enable the custom autograd by default when PythonOp backend support is enabled during build.
-enable_custom_autograd_support(
-    not ortmodule._defined_from_envvar("ORTMODULE_DISABLE_CUSTOM_AUTOGRAD_SUPPORT", 0) and is_torch_interop_default_on()
-)
+enable_custom_autograd_support(is_torch_interop_default_on())
```

Comment:
Maybe `is_torch_interop_default_on` should be imported at `options.py` as part of the options API instead of being a private property at `onnxruntime.capi._pybind_state` namespace?